app/Config/core.php

```diff
@@ -359,7 +359,7 @@
 }
 
 // Prefix each application on the same server with a different string, to avoid Memcache and APC conflicts.
-$prefix = 'myapp_';
+$prefix = Inflector::slug(APP);
```

Comment:
app

Reply:
Please dont comment on a 5+ year old ticket and instead verify this on the current branches and open an issue or PR where needed instead.